RecoLocalCalo/EcalRecProducers/python/ecalUncalibRecHitPhase2Portable_cfi.py

```diff
@@ -0,0 +1,2 @@
+import  RecoLocalCalo.EcalRecProducers.ecalUncalibRecHitPhase2WeightsProducerPortable_cfi as _mod
+ecalUncalibRecHitPhase2Portable = _mod.ecalUncalibRecHitPhase2WeightsProducerPortable.clone()
```

Comment:
What is the purpose of this file? I mean, is it useful to have both `ecalUncalibRecHitPhase2WeightsProducerPortable_cfi` and `ecalUncalibRecHitPhase2Portable_cfi`?